tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "module": "es2020",
```

Comment:
We should compile to CommonJS module system. You may run into same compatibility issues with consumers otherwise as I did with SDK.

Reply:
This is only for selenium right now, but that's a good point we could start using typescript for WebScience, I'll make the output commonjs.

Reply:
Oh actually the recommended config we're extending is commonjs so I think this is OK.

Reply:
🎉